react/features/base/toolbox/components/AbstractAudioMuteButton.js

```diff
@@ -23,11 +24,13 @@ export default class AbstractAudioMuteButton<P: Props, S: *>
      * @returns {void}
      */
     _handleClick() {
-        const { handleClick } = this.props;
+        const { handleClick, notifyMode } = this.props;
 
         if (handleClick) {
             handleClick();
+        }
```

Comment:
shouldn't `handleClick` be also prevented from executing?

Reply:
handleClick only does the API notify part ( if defined)